pandas/tests/arrays/sparse/test_array.py

```diff
@@ -658,12 +658,16 @@ def test_getslice_tuple(self):
         dense = np.array([np.nan, 0, 3, 4, 0, 5, np.nan, np.nan, 0])
 
         sparse = SparseArray(dense)
-        res = sparse[4:,]  # noqa: E231
+        res = sparse[
+            4:,
+        ]  # noqa: E231
```

Comment:
this is pretty gnarly.  is there an alternative?

Reply:
Seems related to: https://github.com/psf/black/issues/1139

Reply:
Is there a way to turn off formatting for a single line? (something like `res = sparse[4:, ]  # fmt: off`, but from their README it seems this only works for blocks)

Reply:
I was surprised that black doesn't support this - I upvoted -> https://github.com/psf/black/issues/790

I suspect they will revert this  in https://github.com/psf/black/issues/1139